### ISSUE TITLE
Back button in Toolbar really small clickable part (lot of misses)

### ISSUE THREAD

**Issue by ferrannp** [react-navigation/react-navigation#5518]
### Current Behavior

- If you click the back button in the toolbar, it is really easy to miss. I saw several people missing (because they usually click on the left corner).

### Expected Behavior

- Open any native app like GMail, the icon is much more easy to click (all is clickable, like much more padding). I never missed in a native app.

### How to reproduce

Just create a stack navigator with two screens and navigate to a screen (so it has a back button).

My own button for `HeaderRight`:
![screenshot 2019-01-24 at 15 55 50](https://user-images.githubusercontent.com/774577/51686942-afe34f80-1ff1-11e9-9de5-e849a6a720c8.png)

Default back button from `react-navigation` that is hard to click:
![screenshot 2019-01-24 at 15 56 20](https://user-images.githubusercontent.com/774577/51686945-b2de4000-1ff1-11e9-9c8d-9fb94a5a11ab.png)

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 3.0.8
| react-native     | 0.57.8
| node             | 10.11.0
| npm or yarn      | 6.5.0

**Comment by brentvatne** [comment]
yup, I was planning on changing it by moving to react-native-gesture-handler and using a BorderlessButton, but the rippleColor prop currently doesn't work if the background is transparent (https://github.com/kmagiera/react-native-gesture-handler/issues/425) and the `hitSlop` prop on the button didn't work for me either. if you can repro `hitSlop` issues on BorderlessButton and post an issue to react-native-gesture-handler that'd be very helpful!

**Comment by osdnk** [comment]
Fixed with https://github.com/kmagiera/react-native-gesture-handler/pull/400

**Comment by pzatorski** [comment]
Thanks a lot @osdnk
